Jahia/luxe-jahia-demo
# Backlog 22550 npm module organisation

## Description

Moved files according to the new specs
https://jira.jahia.org/browse/BACKLOG-22550

Note : the new "vendors" folder in the spec is actually not a vendors folder because it contains the client js, so we keep the javascript name

Done : 
   -  Moved locales and javascript to the new public folder
   - Moved all the config files of the module to the configuration folder
   - Updated static resources in package.json

## Test 

Check that the files are in the good places according to the specs and then : 


1) Build and deploy the project
2) Import it in Jahia (with the lattest version of npm-module-engin + serverSettings) and check that there is no errors
3) Check that everything works in jahia :
- content-type icons are visible when adding content to a page
- template preview image is visible when creating a site
-  locales are working
-  Logo and favicon are visible
-  Client JS is working when logging
-  content-editor-forms is working (adding an image to an agency is possible)
